data/p3/prepare_p3.py

```diff
@@ -0,0 +1,439 @@
+    'social_i_qa_Show_choices_and_generate_answer',
+    'social_i_qa_Show_choices_and_generate_index',
+    'trec_fine_grained_ABBR',
+    'trec_fine_grained_ABBR_context_first',
+    'trec_fine_grained_DESC',
+    'trec_fine_grained_DESC_context_first',
+    'trec_fine_grained_ENTY',
+    'trec_fine_grained_HUM',
+    'trec_fine_grained_HUM_context_first',
+    'trec_fine_grained_LOC',
+    'trec_fine_grained_LOC_context_first',
+    'trec_fine_grained_NUM',
+    'trec_fine_grained_NUM_context_first',
+    'trec_fine_grained_open',
+    'trec_fine_grained_open_context_first',
+    'trec_pick_the_best_descriptor',
+    'trec_trec1',
+    'trec_trec2',
+    'trec_what_category_best_describe',
+    'trec_which_category_best_describes',
+    'wiki_bio_comprehension',
+    'wiki_bio_guess_person',
+    'wiki_bio_key_content',
+    'wiki_bio_what_content',
+    'wiki_bio_who',
+    'wiki_hop_original_choose_best_object_affirmative_1',
+    'wiki_hop_original_choose_best_object_affirmative_2',
+    'wiki_hop_original_choose_best_object_affirmative_3',
+    'wiki_hop_original_choose_best_object_interrogative_1',
+    'wiki_hop_original_choose_best_object_interrogative_2',
+    'wiki_hop_original_explain_relation',
+    'wiki_hop_original_generate_object',
+    'wiki_hop_original_generate_subject',
+    'wiki_hop_original_generate_subject_and_object',
+    'wiki_qa_Decide_good_answer',
+    'wiki_qa_Direct_Answer_to_Question',
+    'wiki_qa_Generate_Question_from_Topic',
+    'wiki_qa_Is_This_True_',
+    'wiki_qa_Jeopardy_style',
+    'wiki_qa_Topic_Prediction_Answer_Only',
+    'wiki_qa_Topic_Prediction_Question_Only',
+    'wiki_qa_Topic_Prediction_Question_and_Answer_Pair',
+    'wiki_qa_automatic_system',
+    'wiki_qa_exercise',
+    'wiki_qa_found_on_google',
+    'wiqa_does_the_supposed_perturbation_have_an_effect',
+    'wiqa_effect_with_label_answer',
+    'wiqa_effect_with_string_answer',
+    'wiqa_what_is_the_final_step_of_the_following_process',
+    'wiqa_what_is_the_missing_first_step',
+    'wiqa_what_might_be_the_first_step_of_the_process',
+    'wiqa_what_might_be_the_last_step_of_the_process',
+    'wiqa_which_of_the_following_is_the_supposed_perturbation',
+    'xsum_DOC_boils_down_to_simple_idea_that',
+    'xsum_DOC_given_above_write_one_sentence',
+    'xsum_DOC_how_would_you_rephrase_few_words',
+    'xsum_DOC_tldr',
+    'xsum_DOC_write_summary_of_above',
+    'xsum_article_DOC_summary',
+    'xsum_college_roommate_asked_DOC_so_I_recap',
+    'xsum_read_below_DOC_write_abstract',
+    'xsum_summarize_DOC',
+    'xsum_summarize_this_DOC_summary',
+    'yelp_review_full_based_on_that',
+    'yelp_review_full_format_rating',
+    'yelp_review_full_format_score',
+    'yelp_review_full_format_star',
+    'yelp_review_full_on_a_scale',
+    'yelp_review_full_so_i_would',
+    'yelp_review_full_this_place'
+]
+
+# Optonally download all first
+# for task_name in TZERO_TASK_LIST:
+#     ds = load_dataset("bigscience/P3", task_name)
```

Comment:
Why would that be useful?

Reply:
To run the downloading on a CPU + Internet instance & the preprocessing e.g. on one without internet